src/scripts/filter.sh

```diff
@@ -96,7 +96,7 @@ done
 # If auto-detecting is enabled (or modules aren't set), check for configs in .circleci/.
 if [ "$SH_AUTO_DETECT" -eq 1 ] || [ "$SH_MODULES" = "" ]; then
     # We need to determine what the modules are, ignoring SH_MODULES if it is set.
-    SH_MODULES="$(find .circleci/ -type f -name "*.yml" | grep -oP "(?<=.circleci/).*(?=.yml)" | grep -v config | sed "s@${SH_ROOT_MODULE}@.@")"
+    SH_MODULES="$(find .circleci/ -type f -name "*.yml" | grep -oP "(?<=.circleci/).*(?=.yml)" | grep -v "^/config$" | sed "s@${SH_ROOT_MODULE}@.@")"
```

Comment:
Hmm, should this be `"^(config)$`? Not sure that makes any difference.

Reply:
Oh I see, you've included examples above, I can't read haha.

Reply:
Ohh I see, the `grep -v config` is filtering all filenames that contain the word `config`, yeah that's blatantly wrong. We can merge this.